src/components/drop-down-options/drop-down-options.test.tsx

```diff
@@ -24,7 +24,7 @@ describe('Button', () => {
 
     rerender(<DropDownOptions show={false} options={TEST_OPTIONS_SINGLE} onOptionSelect={noop} />);
     await waitForElementToBeRemoved(() => screen.queryByText(TEST_OPTIONS_SINGLE_VALUES[0]), {
-      timeout: 500, // wrt `Fade` having transition duration of 0.2 (200ms)
+      timeout: 1000, // wrt `Fade` having transition duration of 0.2 (200ms)
```

Comment:
😕 async code be like

Reply:
:sob:

Reply:
yeah I'm still not convinced this is a good pattern, it's highly dependent on the frame-rate (so however much cpu load the test container has) due to `requestAnimationFrame`

i think i'm gonna audit this test and rewrite it better in another PR